tests/lowering/tensor_manipulation/test_slice.py

```diff
@@ -0,0 +1,38 @@
+import torch
+import torch_ttnn
+import pytest
+import ttnn
+
+
+class SliceModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, input, start, end):
+        return input[:, :, start:end]
+
+
+@pytest.mark.parametrize(
```

Comment:
Sample data from Bert. Can you please codify in the test?

**case 1**
Tensor<[1, 1, 1, 256]> self = ?, int<> dim = 3, Optional[int]<> start = 0, Optional[int]<> end = -1

**case 2**
Tensor<[1, 256]> self = ?, int<> dim = 0, Optional[int]<> start = 0, Optional[int]<> end = -1

**case 3**
Tensor<[1, 512]> self = ?, int<> dim = 0, Optional[int]<> start = 0, Optional[int]<> end = -1

**case 4**
Tensor<[1, 512]> self = ?, int<> dim = 1, Optional[int]<> start = 0, Optional[int]<> end = 256

Reply:
https://github.com/tenstorrent/tt-metal/pull/12469

this change will make these work with tiled.